torchci/pages/tts/[repoOwner]/[repoName]/[branch]/[[...page]].tsx

```diff
@@ -266,10 +274,18 @@ function GranularityPicker({
 }
 
 export default function Page() {
+  const router = useRouter();
+  const branch =
+    router.query.branch === undefined ? "master": router.query.branch as string;
```

Comment:
you might be able to do something like `router.query.branch ?? master` instad of this ternary